core/src/kvs/lq_structs.rs

```diff
@@ -93,9 +93,9 @@ pub(crate) struct LqEntry {
 /// For example, live query IDs need to be tracked by websockets so they are closed correctly on closing a connection
 #[derive(Debug)]
 #[cfg_attr(test, derive(PartialEq, Clone))]
+#[allow(dead_code)]
 pub(crate) enum TrackedResult {
 	LiveQuery(LqEntry),
```

Comment:
Tbh I dont know really why this isnt used. When it was introduced it definitely was used. We can keep it in codebase, but will check it more thanks

Reply:
It is being used. It's only that `dead_code` was previously only allowed for the `KillQuery` variant but `LiveQuery` also generates unused code warnings for some feature flags. Since this enum only has two variants, I allowed dead code on the enum itself.